src/ckb-daemon/keymap.h

```diff
@@ -64,7 +64,7 @@ extern const key keymap[N_KEYS_EXTENDED];
 // Translates input from HID to a ckb input bitfield.
 // Use positive endpoint for non-RGB keyboards, negative endpoint for RGB
 void hid_kb_translate(unsigned char* kbinput, int endpoint, int length, const unsigned char* urbinput);
-void hid_mouse_translate(unsigned char* kbinput, short* xaxis, short* yaxis, int endpoint, int length, const unsigned char* urbinput, ushort fwversion);
+void hid_mouse_translate(unsigned char* kbinput, short* xaxis, short* yaxis, int endpoint, int length, const unsigned char* urbinput, void* context);
```

Comment:
Shouldn't this be a `struct usbdevice*`?

Reply:
I couldn't figure out how to include the header properly. You are welcome to fix it properly.

Reply:
Yeah, it seems that that would introduce a circular dependency.